static/elements/chromedash-textarea.js

```diff
@@ -0,0 +1,58 @@
+
+import SlTextarea from '@shoelace-style/shoelace/dist/components/textarea/textarea.js';
+
+class ChromedashTextarea extends SlTextarea {
+  static get properties() {
+    return {
+      ...super.properties,
+      chromedash_single_pattern: {type: String},
+      chromedash_split_pattern: {type: String},
+    };
+  }
+
+  constructor() {
+    super();
+    this.originalCheckValidity = null;
+  }
+
+  updated() {
+    if (!this.input) {
+      return;
+    }
+    if (!this.originalCheckValidity) {
+      // Set up custom validation.
+      this.originalCheckValidity = this.input.checkValidity;
+      this.input.checkValidity = () => {
+        const value = this.input.value;
+        // First do default validity check.
+        if (this.originalCheckValidity && !this.originalCheckValidity.apply(this.input)) {
+          return false;
+        }
+        // If multiple, then use chrome
```

Comment:
This comment is incomplete.

Reply:
Yup, writing the comment got me to think about generalizing the process a bit... and I forgot to return to finish the thought.